docusaurus.config.js

```diff
@@ -146,7 +146,7 @@ const versions = require('./versions.json');
       navbar: {
         title: 'Apache DevLake',
         logo: {
-          alt: 'apache-devlake',
+          alt: 'logo',
```

Comment:
why can you not use `devlake` as a keyword?

Reply:
@mintsweet Because I want to show the other image. So I removed `devlake` from the logo's alt and added the `devlake` to the DORA image's alt.

Reply:
Get it.